package.json

```diff
@@ -17,51 +17,50 @@
 	"version": "1.0.0",
 	"main": "./dist/index.js",
 	"engines": {
-		"node": ">=7"
+		"node": ">=8"
 	},
 	"scripts": {
+
 		"start": "node dist/index.js",
-		"prepublishOnly": "npm run build:transpile",
+		"cli": "node dist/cli.js",
 		"check-deps": "npm-check;exit 0",
-		"lint": "eslint --fix src",
-		"test:unit:base": "cross-env NODE_ENV=test nyc -x test-fixtures -x test/**/*.spec.js mocha --require babel-core/register",
-		"test:unit": "npm run test:unit:base -- test/**/*.spec.js",
-		"test": "npm run lint && npm run test:unit && npm run coverage",
-		"clean": "rimraf build dist",
-		"coverage": "nyc check-coverage --per-file",
-		"build:transpile": "babel src --source-maps --out-dir=dist --minified",
-		"build": "npm run build:transpile && npm run check-deps",
-		"watch:test": "cross-env DEBUG=1 NODE_ENV=debug nodemon --watch src --exec 'npm test && babel-node src/index.js'",
-		"watch:build": "cross-env DEBUG=1 NODE_ENV=debug nodemon --watch src --exec 'babel-node src/index.js'",
-		"publish-to-npm": "npm install && npm test && npm run coverage && npm publish --access=public"
+		"lint": "eslint src",
+		"lint:dev": "eslint --fix src",
+		"test:base": "cross-env NODE_ENV=test mocha --require @babel/register",
+		"test": "npm run lint && npm run test:base -- src/**/*.spec.js",
+		"test:dev": "npm run lint:dev && npm run test:base -- src/**/*.spec.js",
+		"build": "babel src --source-maps --delete-dir-on-start --out-dir=dist",
+		"build:dev": "npm run build:transpile && npm run check-deps",
+		"watch": "cross-env DEBUG=1 NODE_ENV=debug nodemon --watch src --exec 'npm run test:dev'"
 	},
 	"dependencies": {
-		"@natlibfi/marc-record": "^3.0.0",
-		"@natlibfi/marc-record-validate": "^4.0.0",
-		"@natlibfi/marc-record-validators-melinda": "^5.1.1",
-		"@natlibfi/melinda-record-import-commons": "^1.5.0",
+		"@natlibfi/marc-record": "^4.0.0",
+		"@natlibfi/marc-record-validate": "^5.0.0",
+		"@natlibfi/marc-record-validators-melinda": "^7.0.4",
+		"@natlibfi/melinda-commons": "^7.0.1",
+		"@natlibfi/melinda-record-import-commons": "6.0.2",
+		"get-stream": "^5.1.0",
 		"langs": "^2.0.0",
-		"marc-record-js": "git+https://github.com/natlibfi-arlehiko/marc-record-js.git",
-		"metadata-parser": "^1.0.1",
-		"moment": "^2.22.2",
-		"node-mocks-http": "^1.7.3"
+		"moment": "^2.22.2"
 	},
 	"devDependencies": {
-		"babel-cli": "^6.26.0",
-		"babel-eslint": "^8.2.6",
-		"babel-preset-env": "^1.6.1",
-		"babel-register": "^6.26.0",
-		"chai": "^4.1.2",
+		"@babel/cli": "^7.2.0",
+		"@babel/core": "^7.2.0",
+		"@babel/node": "^7.2.0",
+		"@babel/preset-env": "^7.2.0",
+		"@babel/register": "^7.6.2",
+		"babel-eslint": "^10.0.3",
+		"chai": "^4.2.0",
 		"chai-as-promised": "^7.1.1",
-		"cross-env": "^5.2.0",
-		"eslint": "^5.9.0",
-		"eslint-config-xo": "^0.25.0",
-		"eslint-plugin-import": "^2.14.0",
-		"mocha": "^5.2.0",
-		"nodemon": "^1.18.6",
+		"cross-env": "^6.0.0",
+		"eslint": "^6.4.0",
+		"eslint-config-xo": "^0.26.0",
```

Comment:
Replace with the following: `>=0.26.0 <1.0.0` (Otherwise minor version updates are not applied)